### ISSUE TITLE
Properly fix W/A from #240, commit 661308bd7 - [test_conv_for_implicit_gemm] Disable config that fails often.

### ISSUE THREAD

**Issue by atamazov** [ROCm/MIOpen#1167]
Commit that introduces the W/A: https://github.com/ROCmSoftwarePlatform/MIOpen/commit/661308bd74d96c7ce330c94394b0a43a2bd6fa21

Where it resides now: https://github.com/ROCmSoftwarePlatform/MIOpen/blob/2d7b2dd76464ac0fe052118718c4ab217341d25b/test/CMakeLists.txt#L723

Let's revert the W/A, check if problem still persists and provide a proper fix.

**Comment by junliume** [comment]
Let's see what might or might not pass with this WA reverted:
http://micimaster.amd.com/blue/organizations/jenkins/MLLibs%2FMIOpen/detail/fix_%231167/1/pipeline/

**Comment by ppanchad-amd** [comment]
@atamazov, Is this ticket still relevant?

**Comment by atamazov** [comment]
@ppanchad-amd Yes. The W/A was silently removed in #2513, however, in test/gtest/conv_for_implicit_gemm.cpp, we still need to re-enable the config mentioned in the description of this issue.

/cc @ozturkosu @junliume

**Comment by JonathanLichtnerAMD** [comment]
Closing since MIOpen is being migrated to [ROCm/rocm-libraries](https://github.com/ROCm/rocm-libraries).  Please re-open there if you believe this is still an issue.
